volkszaehler/__init__.py

```diff
@@ -31,9 +34,11 @@ async def get_data(self):
         try:
             with async_timeout.timeout(5, loop=self._loop):
                 response = await self._session.get(self.url)
+                responsenow = await self._session.get(self.urlnow)
```

Comment:
This is a second API call. According to the [Volkzaehler's API documentation](https://wiki.volkszaehler.org/development/api/reference) the default is `(jetzt - 24h)`. Thus the API call that is made contains the data for last 24 h.

At first glance (or what I can see in the data from the demo instance) it looks like `.json?from=now` provides the same information as the current available API call. As `now` is ignored by PHP and the API falls back to the default.

The demo instance would be limited and not act like a real setup.